tests/integration/Paginator/Adapter/RepositoryCest.php

```diff
@@ -0,0 +1,70 @@
+<?php
```

Comment:
Will need to refactor this a bit after we add it to the main repo. I will generate test stubs to split all the methods and make tests shorter. a todo for later.

Reply:
No worries. In future, would you rather we generate the stubs when adding new classes? If so, how do we do that?

Reply:
Those are generated with `zephir stubs`